docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -0,0 +1,352 @@
+
+# Service is up, load balancer is created
+$ kubectl get services
+NAME               TYPE           CLUSTER-IP   EXTERNAL-IP      PORT(S)                        AGE
+teleport-cluster   LoadBalancer   10.4.4.73    104.199.126.88   443:31204/TCP,3026:32690/TCP   89s
+
+# Save the pod IP. If the IP is not available, check the pod and load balancer status.
+$ MYIP=$(kubectl get services teleport-cluster -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+$ echo $MYIP
+192.168.2.1
+```
+</TabItem>
+<TabItem label="Enterprise">
+```bash
+# Set kubectl context to the namespace to set some typing
+$ kubectl config set-context --current --namespace=teleport-cluster-ent
+
+# Service is up, load balancer is created
+$ kubectl get services
+NAME                   TYPE           CLUSTER-IP   EXTERNAL-IP      PORT(S)                        AGE
+teleport-cluster-ent   LoadBalancer   10.4.4.73    104.199.126.88   443:31204/TCP,3026:32690/TCP   89s
+
+# Save the pod IP. If the IP is not available, check the pod and load balancer status.
+$ MYIP=$(kubectl get services teleport-cluster-ent -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+$ echo $MYIP
+192.168.2.1
+```
+</TabItem>
+</Tabs>
+
+
+Set up two `A` DNS records - `tele.example.com` for UI and `*.tele.example.com`
+for web apps using [application access](./application-access.md).
+
+<Tabs>
+<TabItem label="Google DNS">
+```bash
+$ MYZONE="myzone"
+$ MYDNS="tele.example.com"
+
+$ gcloud dns record-sets transaction start --zone="${MYZONE?}"
+$ gcloud dns record-sets transaction add ${MYIP?} --name="${MYDNS?}" --ttl="30" --type="A" --zone="${MYZONE?}"
+$ gcloud dns record-sets transaction add ${MYIP?} --name="*.${MYDNS?}" --ttl="30" --type="A" --zone="${MYZONE?}"
+$ gcloud dns record-sets transaction describe --zone="${MYZONE?}"
+$ gcloud dns record-sets transaction execute --zone="${MYZONE?}"
+```
+</TabItem>
+<TabItem label="AWS Route 53">
+```bash
+# Tip for finding AWS zone id by the domain name.
+$ MYZONE_DNS="example.com"
+$ MYZONE=$(aws route53 list-hosted-zones-by-name --dns-name=${MYZONE_DNS?} | jq -r '.HostedZones[0].Id' | sed s_/hostedzone/__)
+
+$ MYDNS="tele.example.com"
+
+# Create a JSON file changeset for AWS.
+$ jq -n --arg ip ${MYIP?} --arg dns ${MYDNS?} '{"Comment": "Create records", "Changes": [
+  {"Action": "CREATE", "ResourceRecordSet": {"Name": $dns, "Type": "A", "TTL": 300, "ResourceRecords": [{ "Value": $ip}]}},
+  {"Action": "CREATE", "ResourceRecordSet": {"Name": ("*." + $dns), "Type": "A", "TTL": 300, "ResourceRecords": [{ "Value": $ip}]}}
+  ]}' > myrecords.json
+
+# Review records before applying.
+$ cat myrecords.json | jq
+# Apply the records and capture change id
+$ CHANGEID=$(aws route53 change-resource-record-sets --hosted-zone-id ${MYZONE?} --change-batch file://myrecords.json | jq -r '.ChangeInfo.Id')
+
+# Verify that change has been applied
+$ aws route53 get-change --id ${CHANGEID?} | jq '.ChangeInfo.Status'
+"INSYNC"
+```
+</TabItem>
+</Tabs>
```

Comment:
In future maybe we could describe how to set up [`external-dns`](https://github.com/bitnami/charts/tree/master/bitnami/external-dns) to make this part automatic. I'm keen not to try and start teaching people how to use Kubernetes/Helm, though.

Reply:
could be, yes, I saw DNS automation in K8s, but it was quite complex for a getting started guide